<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="suba-pc" failures="1" tests="1" name="Default test" time="14.118" errors="0" timestamp="2024-04-30T20:59:11 EDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="12.06">
    <failure type="java.lang.AssertionError" message="Actual message does not contain expected message expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Actual message does not contain expected message expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at commonUtilities.CommonUtils.getAssertionEqualsCheck(CommonUtils.java:43)
at pageObjects.BatchPage.verifySuccessMessage(BatchPage.java:347)
at stepDefinitions.AddNewBatch_SD.admin_should_see_the_successfull_message(AddNewBatch_SD.java:111)
at âœ½.Admin should see the Successfull message(file:///C:/Users/subas/EclipseLMSPractice/Team15_SeleniumTribe_LMSUIHackathon_Apr24/src/test/resources/Features/D_BatchModule/02_AddNewbatch.feature:47)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
